<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="widget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout" columnstretch="50,50,80,50,0">
      <item row="10" column="0" colspan="5">
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="4">
       <widget class="QLineEdit" name="lineEdit">
        <property name="inputMask">
         <string/>
        </property>
        <property name="placeholderText">
         <string>검색할 도서관 사이트 링크 입력</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0" colspan="5">
       <widget class="QTreeWidget" name="treeWidget">
        <column>
         <property name="text">
          <string>검색 결과</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>서명</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>저자</string>
         </property>
        </column>
        <item>
         <property name="text">
          <string>결과 없음</string>
         </property>
         <item>
          <property name="text">
           <string>서명 서명</string>
          </property>
          <property name="text">
           <string>테스트 서명</string>
          </property>
          <property name="text">
           <string>테스트 저자</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>서명 서명2</string>
          </property>
          <property name="text">
           <string>테스트2 서명</string>
          </property>
          <property name="text">
           <string>테스트2 저자</string>
          </property>
         </item>
        </item>
        <item>
         <property name="text">
          <string>저자 일치</string>
         </property>
         <item>
          <property name="text">
           <string>서명 서명</string>
          </property>
          <property name="text">
           <string>테스트 서명</string>
          </property>
          <property name="text">
           <string>테스트 저자</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>서명 서명2</string>
          </property>
          <property name="text">
           <string>테스트 서명</string>
          </property>
          <property name="text">
           <string>테스트 저자</string>
          </property>
         </item>
        </item>
        <item>
         <property name="text">
          <string>저자 불일치</string>
         </property>
         <item>
          <property name="text">
           <string>서명 서명</string>
          </property>
          <property name="text">
           <string>테스트 서명</string>
          </property>
          <property name="text">
           <string>테스트 저자</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>서명 서명</string>
          </property>
          <property name="text">
           <string>테스트 서명</string>
          </property>
          <property name="text">
           <string>테스트 저자</string>
          </property>
         </item>
        </item>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>진행 상황 실시간 조회</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QRadioButton" name="radioButton_2">
        <property name="text">
         <string>멀티 프로세싱</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="5">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>엑셀 파일 (.xlsx)을 프로그램으로 드래그 해주세요</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QComboBox" name="comboBox"/>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="radioButton">
        <property name="text">
         <string>싱글 프로세싱</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="4">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>시작</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>기관 수정/추가</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
